smithy.yaml

```diff
@@ -0,0 +1,12 @@
+project: dart
+language: dart
+
+# dart 1.24.2
+runner_image: drydock-prod.workiva.net/workiva/smithy-runner-generator:179735
```

Comment:
Do we have smithy turned on for this repo?

Reply:
dependency tracking for Rosie

Reply:
ah, ya, asked smithy people about that on friday and didn't get any response. I'll pester them again...

Reply:
Waiting on ATLAS-23057

Reply:
done